extra/docs_landing.html

```diff
@@ -63,9 +63,10 @@ <h2>Learn about Elastic</h2>
    </p>
    </div>
    <div class="col-md-6 col-12">
-    <iframe class="vidyard_iframe" src="https://play.vidyard.com/qAS1W24928bBrc2sv5f5Ti?v=4.3" width="448" height="252" scrolling="no" frameborder="0" allowfullscreen="" /></iframe>
+     <div class="embed-responsive embed-responsive-16by9">
+       <iframe class="embed-responsive-item vidyard_iframe" src="https://play.vidyard.com/qAS1W24928bBrc2sv5f5Ti?v=4.3&amp;" scrolling="no" frameborder="0" allowfullscreen=""></iframe>
```

Comment:
@dedemorton I might have copied the url incorrectly (did it quickly) so you can likely drop the `&amp; `after `v=4.3` in the video URL like you had in your original example. FWIW!

Reply:
Thanks! I wasn't sure, but it looked like someone copy/pasted and got some ascii characters. :-)